include/fast_float/slow.h

```diff
@@ -0,0 +1,420 @@
+#ifndef FASTFLOAT_SLOW_H
```

Comment:
I guess 'slow' is fine, but it is not the best name. I do not mind having something called 'slow' but I'd like to have you thoughts on whether there could be a better name for  your approach...? Note that we can need to refer to it in other context and saying "well, we use the slow approach" is not... always great.

Reply:
Sure, I can rename it. It's a digit-comparison algorithm, so maybe `digitcmp` is better?

Reply:
@Alexhuszagh It is your approach so you get to name it. I personally like things that I can pronounce so, digit-comparison or `digit_comparison` sounds good?

Think about this... you are in a meeting and you want to refer to "this"... now, it helps if you can just speak it out.

(I realize that this is a minor point.)

Reply:
> @Alexhuszagh It is your approach so you get to name it. I personally like things that I can pronounce so, digit-comparison or `digit_comparison` sounds good?
> 
> Think about this... you are in a meeting and you want to refer to "this"... now, it helps if you can just speak it out.
> 
> (I realize that this is a minor point.)

That sounds great, I'll implement all these recommendations then.